### DISCUSSION TITLE
using uSync.ContentEdition to migrate from Umbraco 7/8 to Umbraco 10.x

### DISCUSSION THREAD

**Discussion by jesperordrup** [KevinJump/uSync#415] [General]
Hi Kevin, 

Is it possible to export from 7/8 to 10 with any level of success? 

thx as always for uSync
best
Jesper

**Comment by tormnator** [comment]
I'm not the authority on this, but I was unable to do what you're asking. Instead I ended up using built in and third party migrations to bring the database via U8 and U9 to Umbraco 10 in a temporary site, then using uSync to move data from the temporary site to the final site (which contained migrated and new .NET 6 compatible code).

**Reply by jesperordrup** [reply]
Thanks @tormnator. That's good input. This ensures a "clean" db which is exactly what I'm after. 

Thanks 🤞
